guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc

```diff
@@ -152,7 +179,6 @@ To check this, navigate to *Infrastructure* > *{SmartProxy}s* and verify if the
 
 In the {Project} web UI, navigate to *Infrastructure* > *Subnets*, select a subnet, click the {SmartProxy}s tab, and select the *Discovery Proxy* that you want to use. Perform this for each appropriate subnet.
 
-
```

Comment:
I am dropping those extra two lines before major chapters - it drives me crazy :-)